nimble/host/syscfg.yml

```diff
@@ -104,6 +104,7 @@ syscfg.defs:
         value: 0
         restrictions:
             - 'BLE_SM_SC_LVL == 4 if 1'
+            - BLE_SM_MITM
```

Comment:
should this also force BLE_SM_SC:1  and BLE_SM_LEGACY:0 ?

Reply:
I haven't though about it, but yes, it makes sense - how could we force SC level on legacy? I'll update this PR